src/pages/Replays/index.js

```diff
@@ -19,8 +19,13 @@ const sortReplays = (replayData, current) => {
   return beggining.concat(end);
 };
 
-const Replays = () => {
-  const [current, setCurrent] = useState(0);
+const Replays = props => {
+  let replayId = 0;
+  if (props.match.params.replayId != null) {
```

Comment:
if you're checking for a truthy value explicitly
should be if(props.match.params.replayId){...}